examples/syscall.c

```diff
@@ -23,13 +23,12 @@
 #include <linux/sched.h>
 #include <linux/uaccess.h>
 
-/* The in-kernel calls to the ksys_close() syscall were removed in Linux v5.11+.
- */
-#if (LINUX_VERSION_CODE < KERNEL_VERSION(5, 11, 0))
-#include <linux/syscalls.h> /* ksys_close() wrapper for backward compatibility */
-#define close_fd ksys_close
-#else
-#include <linux/fdtable.h> /* For close_fd */
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 7, 0)
+#define KPROBE_LOOKUP 1
+#include <linux/kprobes.h>
```

Comment:
I try your lastest syscall.c and it can work smoothly at 5.11.0-27-generic and 5.4.0-80-generic. However, the page fault issue will also appear at 5.8.0-48-generic on my machine.
I change  `#if (LINUX_VERSION_CODE < KERNEL_VERSION(5, 11, 0))` to `#if (LINUX_VERSION_CODE < KERNEL_VERSION(5, 8, 0))`and it can work without error.

Reply:
I see.
Maybe it should remove because of [Control-flow integrity for the kernel [LWN.net]](https://lwn.net/Articles/810077/) from version > v5.4. I will make some changes.
Thanks.